CMakeLists.txt

```diff
@@ -0,0 +1,125 @@
+project (nice)
```

Comment:
While this might improve the build on Windows, I am worried that it would make the situation worse on other platforms. We would be moving from one bad build system (autotools) to another one which is just as bad (cmake). I would rather leave the build system alone, and just accept patches to improve the `.sln` files. The libnice build changes sufficiently irregularly that it is not much work to maintain the `.sln` files manually.

Could you perhaps provide a patch for the `.sln` files based on the diff between git master and what `cmake` generates? Thanks.